src/components/color-system/examples/extended-color-palette-example.tsx

```diff
@@ -0,0 +1,168 @@
+                    <div class="swatch --color-yellow-light"></div>
+                    <div class="swatch --color-yellow-default">·</div>
+                    <div class="swatch --color-yellow-dark"></div>
+                    <div class="swatch --color-yellow-darker"></div>
+                    <div class="swatch hue">yellow</div>
+                    <div class="swatch --color-amber-lighter"></div>
+                    <div class="swatch --color-amber-light"></div>
+                    <div class="swatch --color-amber-default">·</div>
+                    <div class="swatch --color-amber-dark"></div>
+                    <div class="swatch --color-amber-darker"></div>
+                    <div class="swatch hue">amber</div>
+                    <div class="swatch --color-orange-lighter"></div>
+                    <div class="swatch --color-orange-light"></div>
+                    <div class="swatch --color-orange-default">·</div>
+                    <div class="swatch --color-orange-dark"></div>
+                    <div class="swatch --color-orange-darker"></div>
+                    <div class="swatch hue">orange</div>
+                    <div class="swatch --color-coral-lighter"></div>
+                    <div class="swatch --color-coral-light"></div>
+                    <div class="swatch --color-coral-default">·</div>
+                    <div class="swatch --color-coral-dark"></div>
+                    <div class="swatch --color-coral-darker"></div>
+                    <div class="swatch hue">coral</div>
+                    <div class="swatch --color-brown-lighter"></div>
+                    <div class="swatch --color-brown-light"></div>
+                    <div class="swatch --color-brown-default">·</div>
+                    <div class="swatch --color-brown-dark"></div>
+                    <div class="swatch --color-brown-darker"></div>
+                    <div class="swatch hue">brown</div>
+                    <div class="swatch --color-gray-lighter"></div>
+                    <div class="swatch --color-gray-light"></div>
+                    <div class="swatch --color-gray-default">·</div>
+                    <div class="swatch --color-gray-dark"></div>
+                    <div class="swatch --color-gray-darker"></div>
+                    <div class="swatch hue">gray</div>
+                    <div class="swatch --color-glaucous-lighter has-brightness-label lighter"></div>
+                    <div class="swatch --color-glaucous-light has-brightness-label light"></div>
+                    <div class="swatch --color-glaucous-default has-brightness-label default">
+                        ·
+                    </div>
+                    <div class="swatch --color-glaucous-dark has-brightness-label dark"></div>
+                    <div class="swatch --color-glaucous-darker has-brightness-label darker"></div>
+                    <div class="swatch hue">glaucous</div>
```

Comment:
Lint :/

Reply:
I didn't mean the difference in linebreaks, I mean the difference in css classes. All the others just have two classes (`class="swatch --color-gray-darker"`) but glaucous has four (`class="swatch --color-glaucous-darker has-brightness-label darker"`)?

I'm guessing this is just cruft left over from trying things out, because I couldn't see any difference when I put a column without the extra classes next to the one with…?

Reply:
Ah! This is to do with the y-axis labels… 😬

I'll see if I can maybe solve that in some slightly nicer way…

Reply:
⚡

Reply:
oh yea sorry, that's the y-axis stuff. and your solution is awesome!